index.html

```diff
@@ -75,9 +78,62 @@ <h2 class="tonic1">Tonic</h2> <br>
               <li class="back4">css</li>
               <li class="back4">javascript</li>
             </ul><br>
-            <button type="button" class="button button4">See Project</button>
+            <button type="button" class="button button4 myBtn">See Project</button>
+          </div>
+        </div>
+
+        <!-- --------------------------------------------First Modal--------------------------------------------- -->
```

Comment:
- [ ] Please remove all the comments that look similar to this and run `npx hint .` on your terminal. This should fix the error. HTML comments should be written in this format `<!-- Comment -->`